src/ServiceManagement/Network/Network.Tests/Network.Tests.csproj

```diff
@@ -533,6 +557,30 @@
     <None Include="SessionRecords\Network.Tests.Routes.SetRouteTests\SetRouteWithNullRouteNameInParameters.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Network.Tests.IPForwardingIntegrationTests\SetIPForwardingOnNICSucceeds.json">
```

Comment:
It seems like you added the json files (session records) but didn't add the actual tests.

Need to refresh the file NetworkSecurityGroupIntegrationTests.cs,

as well as add some new files for IPForwarding (IPForwardingIntegrationTests.cs) and Routes (GetEffectiveRouteTableTests.cs)